mimic/test/test_catalog.py

```diff
@@ -0,0 +1,116 @@
+from __future__ import absolute_import, division, unicode_literals
+
+from twisted.trial.unittest import SynchronousTestCase
+
+from mimic.catalog import Endpoint, Entry
+
+
+class CatalogEntry(SynchronousTestCase):
+    """
+    Tests for creating a :class:`Entry` object
```

Comment:
What would you think of modifying the styles of the test docstring to contain assertions instead of descriptions? This can be helpful to other maintainers and users as it should make it clear what the test is intending to test. For example, instead of staying "Absolute mimial creation of :obj:`Entry`", the docstring could say what conditions one should expect to be fulfilled by created a new `Entry` object.